packages/nimble-components/src/table/styles.ts

```diff
@@ -91,18 +106,48 @@ export const styles = css`
         padding-left: 0px;
     }
 
+    .group-row {
+        background: ${sectionBackgroundColor};
+        position: relative;
+        height: calc(${controlHeight} + 2 * ${borderWidth});
+        border-top: calc(2 * ${borderWidth}) solid transparent;
+        box-sizing: border-box;
+        background-clip: padding-box;
+    }
+
+    .group-row::before {
+        content: '';
+        width: 100%;
+        height: ${controlHeight};
+        position: absolute;
+        pointer-events: none;
+        bottom: 0px;
+        background-clip: padding-box;
+    }
+
+    .group-row:hover::before {
+        background: ${fillHoverColor};
+        background-clip: padding-box;
+    }
```

Comment:
Didn't think about it too hard but could most of the none positioning styling be moved into the group-row element's own styles (color, sizing, borders, background, etc)? Applies to the stuff below as well.

If the table needs to override the background color based on it's own attributes (i.e. selection-mode) that makes sense to override from here,